### PR TITLE
Updates readme with additional documentation and fixes

### PR DESCRIPTION
**Pull request by JamesMowery**
Updated a few lines of the documentation just to add clarity and potentially reduce the number of issues moving forward:

- Added a note about the release having been tested on Ubuntu
- Added a note about the `--cache-directory` option
- Added a note about the the magic error and possible solutions
- Added a note about how the full-install command can also be used to update an existing installation
- Fixed a small grammatical error on the glibc error line and streamlined it.

**Note: This is my very first pull request on GitHub. Feedback on how to do this better in the future is welcomed!**

### CODE REVIEW

**Comment by JamesMowery** [comment]
I added a new commit to fix the above comments. https://github.com/Mord3rca/gamma-launcher/pull/114/commits/a1985aed55cf2fcd81f2cccdf325060e405616a5

I have no idea what buttons I need to press in order to resubmit everything!

File: README.md [hunk 1]
```diff
@@ -10,9 +10,13 @@ This is a reimplementation of G.AM.M.A. launcher used for the first setup. You w
...
 By downloading gamma-launcher from the [latest release](https://github.com/Mord3rca/gamma-launcher/releases/latest),
-you can use it without any installation. Everything is self contained in an executable.
+you can use it without any installation. Everything is self contained in an executable. Tested on Ubuntu.
 
-However, if you may have glibc error, in this case, install gamma launcher in a venv.
+Use the `--cache-directory` option to re-use previously downloaded files.
+
+If you experience a glibc error, install gamma launcher in a venv.
+
+If you experience a magic error, prepend `MAGIC=/usr/share/file/misc/magic` and/or `LD_PRELOAD=/usr/lib/libmagic.so` to the `./gamma-launcher` command.
```
[the remark this answers is not in the document]
Comment (JamesMowery) [reply README.md]: Moved these to there own section at the end.

File: README.md [hunk 2]
```diff
@@ -10,9 +10,13 @@ This is a reimplementation of G.AM.M.A. launcher used for the first setup. You w
 ### Using release
 
 By downloading gamma-launcher from the [latest release](https://github.com/Mord3rca/gamma-launcher/releases/latest),
-you can use it without any installation. Everything is self contained in an executable.
+you can use it without any installation. Everything is self contained in an executable. Tested on Ubuntu.
```
[the remark this answers is not in the document]
Comment (JamesMowery) [reply README.md]: Updated the language

**Comment by Mord3rca** [comment]
@JamesMowery The best way to change your history will be to reset your branch on `origin/master` (`git reset origin/master`) and use `git gui` to recreate your commits will adding your diff line by line

**Comment by JamesMowery** [comment]
I've kinda overwhelmed myself a bit with Git. So I'm closing this and starting fresh with a new branch & pull request. Continued over at https://github.com/Mord3rca/gamma-launcher/pull/116

**Pull request closed without merging**